src/app/services/ServiceContainer.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import Button from '../../components/button';
+import NavBar from '../NavBar';
+import ServicesInfoPage from './ServicesInfoPage';
+import ServicesRecapPage from './ServicesRecapPage';
+
+const data = {
```

Comment:
You should be able to use the /taxonomy endpoint for this now instead of hard-coding.

Granted, it's still showing the categories from the NYPL PDF, but I think that's fine. At the very least, it'll be a very strong reminder that we need to map those services already. :sweat_smile: